README.md

```diff
@@ -21,6 +21,34 @@ GPT Engineer is made to be easy to adapt, extend, and make your agent learn how
 - Fast handovers back and forth between AI and human
 - Simplicity, all computation is "resumable" and persisted to the filesystem
 
+## Workflow
+
+**Step 1: Installation**
+- Before you can begin using GPT-Engineer, you need to ensure that it's installed on your system. You can install it using the following command:
+```
+pip install gpt-engineer
+```
+**Step 2: Initial Code Generation**
```

Comment:
I think we also need to set up the OpenAI API key before running any `gpt-engineer` commands

Reply:
Yeah we need to.